model/Core/Vocabularies/ExternalReferenceType.md

```diff
@@ -22,19 +22,34 @@ ExteralReferenceType specifies the type of an external reference.
+- dynamicAnalysisReport: A reference to dynamic analysis report for a package.
 - funding: A reference to funding informaton related to a package.
 - issueTracker: A reference to the issue tracker for a package.
 - mailingList: A reference to the mailing list used by the maintainer for a package.
 - metrics: A reference to metrics related to package such as OpenSSF scorecards.
 - license: A reference to additional license information related to an artifact.
 - other: Used when the type doesn't match any of the other options.
+- product: A reference to product information related to a package.
 - releaseNotes: A reference to the release notes for a package.
 - releaseHistory: A reference to published list of releases for a package.
+- riskAssessment: A reference to risk assessment for a package.
+- runtimeAnalysisReport: A reference to runtime analysis report for a package.
 - securityAdvisory: A reference to a published security advisory (where advisory as defined per ISO 29147:2018) that may affect one or more elements, e.g., vendor advisories or specific NVD entries.
+- securityAdversaryModel: A reference to the security adversary model for a package.
 - securityFix: A reference to the patch or source code that fixes a vulnerability.
 - securityOther: A reference to related security information of unspecified type.
+- securityPenTestReport: A reference to [penetration test](https://en.wikipedia.org/wiki/Penetration_test) report for a package.
+- securityPolicy: A reference to instructions for reporting security vulnerabilities for a package.
+- securityThreatModel: A reference the [security threat model](https://en.wikipedia.org/wiki/Threat_model) for a package.
 - socialMedia: A reference to social media channel for a package.
 - sourceArtifact: A reference to an artifact containing the sources for a package.
-- support: A reference to support channel for a package.
+- staticAnalysisReport: A reference to static analysis report for a package.
+- support: A reference to support channel or information for a package.
 - vcs: A reference to a version control system related to a software artifact.
+- vulnerabilityExploitabilityAssessment: A reference to Vulnerability Exploitability eXchange (VEX) which provides information on whether a product is impacted by a specific vulnerability in an included package and, if affected, whether there are actions recommended to remediate. See also [NTIA VEX one-page](https://ntia.gov/files/ntia/publications/vex_one-page_summary.pdf).
+- vulnerabilityAttestation: A reference to a Vulnerability Disclosure Report (VDR) - a attestation of known vulnerabilities including the analysis and findings describing the impact (or lack of impact) that the reported vulnerability has on a package or product as defined in [NIST SP 800-161](https://csrc.nist.gov/publications/detail/sp/800-161/rev-1/final).
+- quality: A reference to quality information for a package.
```

Comment:
Is this intended for pointing to a formal quality assessment report?   or some of policies that a package may mandate for submissions.

Reply:
Agreed in Security team meeting of May 31st to update it to qualityAssessmentReport: A reference to quality assessment for a package.